api/src/pcapi/core/educational/repository.py

```diff
@@ -977,7 +977,7 @@ def get_paginated_active_collective_offer_template_ids(limit: int, page: int) ->
     return [offer_id for offer_id, in query]
 
 
-def get_banking_information_status_for_booking(booking_id: int) -> CollectiveBookingBankInformationStatus:
+def get_reimbursement_venue_for_booking(booking_id: int) -> offerers_models.Venue:
```

Comment:
Il peut ne pas y avoir de point de remboursement.

```suggestion
def get_reimbursement_venue_for_booking(booking_id: int) -> offerers_models.Venue | None:
```